backend/src/models/user.ts

```diff
@@ -0,0 +1,10 @@
+export type User = {
+  user_id: number;
+  email: string;
+  first_name: string;
+  passwd: string;
+  unallocated_funds: string;
+};
+
+export type CreateUserPayload = Omit<User, 'user_id'>;
```

Comment:
Used for when sending a post request to create a user, the payload has some attributes of user not all. Omit just takes the type and removes the fields that you specify